README.md

```diff
@@ -14,17 +14,20 @@ Add Mobile Center services to your app and collect crash reports and understand
 
 2. **Crashes**: The Mobile Center SDK will automatically generate a crash log every time your app crashes. The log is first written to the device's storage and when the user starts the app again, the crash report will be forwarded to Mobile Center. Collecting crashes works for both beta and live apps, i.e. those submitted to App Store. Crash logs contain viable information for you to help resolve the issue. Crashes uses PLCrashReporter 1.2.1.
 
+3. **Distribute**: Our SDK will let your users install a new version of the app when you distribute it via Mobile Center. With a new version of the app available, the SDK will present an update dialog to the users to either download or ignore the latest version. Once they click "Download", SDK will start the installation process of your application. Note that this feature will `NOT` work if your app is deployed to the app store, if you are developing locally or if the app is a debug build.
```

Comment:
Start with "The Mobile Center SDK .." to match previous paragraphs.

Also you don't "click" on an iPhone.

"Once they click "Download", [the] SDK.." <- missing "the".

"debug build" -> "is running with the debug CONFIGURATION".